streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetadataState.java

```diff
@@ -152,10 +154,16 @@ public String toString(final String indent) {
             return null;
         }
 
-        return getStreamsMetadataForKey(storeName,
-                                        key,
-                                        new DefaultStreamPartitioner<>(keySerializer, clusterMetadata),
-                                        sourceTopicsInfo);
+        if (assignmentVersion >= 4)
```

Comment:
Nit: please add `{` and `}` for all blocks (it's agreed code style) throughout the whole PR.

Reply:
Sure, will make this change.